components/authz-service/server/v2/project_test.go

```diff
@@ -33,6 +33,8 @@ import (
 	"github.com/chef/automate/lib/tls/test/helpers"
 )
 
+const projectLimitForTesting = 10 // DefaultProjectLimit is 300
```

Comment:
This becomes a maintenance issue. Maybe:
```suggestion
const projectLimitForTesting = 10 // DefaultProjectLimit is higher
```